### PR TITLE
9- view a separate page for each category

### PR DESCRIPTION
**Pull request by nasben64**

### CODE REVIEW

File: src/apis/reviews.js [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
-    .then((response) => {
-      return response.data.reviews;
-    })
-    .catch((err) => {
-      console.log(err);
-    });
+export const getReviews = async (searchParams) => {
+  try {
+    let response = [];
+    if (searchParams === "") {
```
Comment (hevdevs) [inline src/apis/reviews.js]: this should work without the need of an if condition (searchParams should be ignored by axios if its value is undefined, give it a go!)

**Review by hevdevs — APPROVED** [review]
Nice work Nasser! good to merge 🎉

File: src/apis/reviews.js (hunk 1 above)
Comment (nasben64) [reply src/apis/reviews.js]: This is how I have done first but I was getting errors when there is no value passed, so I had to add the if condition

Comment (nasben64) [reply src/apis/reviews.js]: I will try it again and tell you what error I am getting

**Pull request merged**